starter_code/models/movie.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+const Schema   = mongoose.Schema;
+const moviesSchema = new Schema({
+    title: String,
+  director: String,
+  stars: [],
```

Comment:
You should to write the word Array, rather than just square brackets. It would also be a good idea to put the title as required because you don't want to save a movie without a title